crates/ruff_python_index/src/multiline_ranges.rs

```diff
@@ -46,7 +46,7 @@ pub(crate) struct MultilineRangesBuilder {
 
 impl MultilineRangesBuilder {
     pub(crate) fn visit_token(&mut self, token: &Token) {
-        if matches!(token.kind(), TokenKind::String | TokenKind::FStringStart) {
+        if matches!(token.kind(), TokenKind::String | TokenKind::FStringMiddle) {
             if token.is_triple_quoted_string() {
                 self.ranges.push(token.range());
             }
```

Comment:
The range should be of `FStringMiddle`, not `FStringStart`.

Reply:
Is this a bug in main too or something you introduced by your refactor?

Reply:
The later, it was introduced by me